api/envoy/extensions/filters/network/zookeeper_proxy/v3/zookeeper_proxy.proto

```diff
@@ -39,4 +40,60 @@ message ZooKeeperProxy {
   //
   // if that is set. If it isn't, ZooKeeper's default is also 1Mb.
   google.protobuf.UInt32Value max_packet_bytes = 3;
+
+  // List of latency thresholds for opcodes to decide the fast/slow responses and emit metrics (used for error budget calculation).
+  //
+  // https://sre.google/workbook/implementing-slos/
+  //
+  // If no latency threshold is set, the fast/slow response counters will always be 0.
```

Comment:
// The fast/slow response stats are emitted only if at least one latency threshold is set.

Reply:
I think the admin interface `/stats` will show all counters include ones with value 0.

Reply:
sure